### PR TITLE
chore: Updating Azure CNI to v1.0.30

### PR DESCRIPTION
**Pull request by jaer-tsun**
**Reason for Change**:

Updates to the latest release of Azure CNI

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jaer-tsun** [comment]
splitting the NPM and CNI version updates

File: packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -83,8 +83,8 @@ function Get-FilesToCacheOnVHD
             "https://acs-mirror.azureedge.net/wink8s/v1.17.0-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://acs-mirror.azureedge.net/cni/azure-vnet-cni-windows-amd64-v1.0.28.zip",
```
Comment (jackfrancis) [inline packer/configure-windows-vhd.ps1]: Let's keep 1.0.28 in the VHD to allow folks to use the VHD who don't want to necessarily update to 1.0.30 or 1.0.29

File: packer/install-dependencies.sh [hunk 2]
```diff
@@ -62,8 +62,8 @@ installGPUDrivers
 echo "  - nvidia-docker2 nvidia-container-runtime" >> ${VHD_LOGS_FILEPATH}
 
 VNET_CNI_VERSIONS="
+1.0.30
 1.0.29
-1.0.28
```
Comment (jackfrancis) [inline packer/install-dependencies.sh]: ditto

File: packer/install-dependencies.sh [hunk 3]
```diff
@@ -317,8 +317,8 @@ for AZURE_NPM_VERSION in ${AZURE_NPM_VERSIONS}; do
 done
 
 AZURE_VNET_TELEMETRY_VERSIONS="
+1.0.30
 1.0.29
-1.0.28
```
Comment (jackfrancis) [inline packer/install-dependencies.sh]: ditto

Comment (jackfrancis) [reply packer/install-dependencies.sh]: Actually, this is for the Azure NPM spec, so let's remove all changes to this array and restrict them to the Azure NPM PR

File: packer/configure-windows-vhd.ps1 (hunk 1 above)
Comment (marosset) [reply packer/configure-windows-vhd.ps1]: +1

Comment (jaer-tsun) [reply packer/configure-windows-vhd.ps1]: added it back

File: packer/install-dependencies.sh (hunk 3 above)
Comment (jaer-tsun) [reply packer/install-dependencies.sh]: reverted npm changes

**Comment by jaer-tsun** [comment]
@jackfrancis is this ready to be merged? looks like all the tests passed

**Pull request merged**